scripts/build_mpc_zokrates_test.zsh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 disable -r time
 
-cargo build --release --example circ
+cargo build --release --features smt,zok_front --example circ
```

Comment:
Hm. The python script should have already built the `circ` binary. Will this cause a rebuild if the global feature list is a strict superset?